lightning/src/chain/channelmonitor.rs

```diff
@@ -155,7 +161,14 @@ pub enum ChannelMonitorUpdateErr {
 /// corrupted.
 /// Contains a human-readable error message.
 #[derive(Debug)]
-pub struct MonitorUpdateError(pub &'static str);
+pub enum MonitorUpdateError {
+	/// While updating the monitor generated an error, still persist the monitor as it has changed
+	/// and needs to be updated on-disk. Contains a human-readable error message.
```

Comment:
Lets call it "developer-readable" since they likely shouldnt be given to end-users :).